website/App.js

```diff
@@ -3,23 +3,60 @@ import * as React from 'react'
 
 const context = require.context('./examples', false, /\.js$/)
 const examples = context.keys().map(k => {
-  const title = startCase(k.replace(/\.js$/, ''))
+  const url = k.replace(/\.js$/, '').slice(2)
+  const title = startCase(url)
   const content = context(k).default
-  return {title, content}
+  return {url, title, content}
 })
 
-export default function App() {
+function renderExample(example) {
   return (
-    <React.StrictMode>
-      <h1>Orama</h1>
-      <div className="grid">
-        {examples.map(example => (
-          <div key={example.title} className="example">
-            <h3>{example.title}</h3>
-            {example.content}
-          </div>
-        ))}
-      </div>
-    </React.StrictMode>
+    <div key={example.title} className="example">
+      <h3>
+        <a href={`#${example.url}`}>{example.title}</a>
+      </h3>
+      {example.content}
+    </div>
```

Comment:
Thoughts on making this a component (`Example`) instead? I prefer it because then the `key` can be specified in a more logical place.

Reply:
Good idea!